### DISCUSSION TITLE
Error: failed to sync fork: fork longer than threshold

### DISCUSSION THREAD

**Discussion by hesen2006cn** [filecoin-project/lotus#7947] [Build and Sync]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

lotus daemon - chain sync

### Lotus Version

```text
v1.11.1
```

### Describe the Bug

lotus sync status

### Logging Information

```text
sync status:
worker 0:
	Base:	[bafy2bzaceckk5knxskvyzbzj4eooewnzka4zjsbsny422sbxj6tkktazd5za2]
	Target:	[bafy2bzacedgyu7fmthajw7gk7r6wd4td5uhapahaho34hyzsca2g6nldfzdcg bafy2bzacecghmgek67oaibxoekurlv3ok75qzwrlpoog5kyuvt6cmlufjwfee bafy2bzaceci5dufxefgiikkdcpyv3ylqf4b4okuaxarn4nmekagzzvjhsinjs] (405453)
	Height diff:	5228
	Stage: error
	Height: 402949
	Elapsed: 7.520208756s
	Error: failed to sync fork: fork longer than threshold
```

### Repo Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

**Comment by Reiers** [comment]
**Hi @hesen2006cn** 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Also checkout: https://lotus.filecoin.io/docs/set-up/chain-management/#lightweight-snapshot

Closing Lotus issue ticket.

**Comment by SgtCoin** [comment]
Hi @hesen2006cn,

My name is Trey, but folks call me Sgt_coin in slack and github. I am one of the support engineers here and wanted to reach out and see what the current status of your lotus node is. Are you still having issues with the fork? If not, how did you correct it?
